test/parallel/test-dns-onlookup.js

```diff
@@ -0,0 +1,69 @@
+// Flags: --expose-internals
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { dnsException } = require('internal/errors');
+const { internalBinding } = require('internal/test/binding');
+const { getaddrinfo, GetAddrInfoReqWrap } = internalBinding('cares_wrap');
+const dns = require('dns');
+const dnsPromises = dns.promises;
```

Comment:
The linter is complaining that this is unused 😄

Reply:
@cjihrig I missed it because I had only checked for single test file with 
"python tools/test.py -J --mode=release parallel/test-dns-onlookup" :relieved: :blush: :blush: